src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="ccf-app mat-app-background">
-  <ccf-header></ccf-header>
+  <ccf-header (logoClicked)="left.resetDrawer(); right.resetDrawer(); filterbox.removeBox()"></ccf-header>
```

Comment:
Refactor this into a method on AppComponent. Since access to the drawers/filters is needed pass them as parameters i.e.
```suggestion
  <ccf-header (logoClicked)="reset(left, right, filterbox)"></ccf-header>
```